src/ApiService/ApiService/Functions/AgentEvents.cs

```diff
@@ -155,7 +155,8 @@ public class AgentEvents {
             // if tasks are running on the node when it reports as Done
             // those are stopped early
             await _context.NodeOperations.MarkTasksStoppedEarly(node, error);
-            await _context.NodeOperations.ToReimage(node, done: true);
+            // result ignored: not used after this point
+            _ = await _context.NodeOperations.ToReimage(node, done: true);
```

Comment:
I propose we encapsulate this pattern in an extension method called ignore 
soemthing like 
``` C#
static class TaskExtensions
{
	public static async Task Ignore<T>(this Task<T> task) {
		_ = await task;
	}
}
```

and this call can become. The method name shows the intent so no need to add the comment 
``` suggestion
            await _context.NodeOperations.ToReimage(node, done: true).Ignore();
```

Reply:
Good idea. This will make it easier to find and fix things in future as well.

Reply:
At the moment I'm only using this helper in cases where we need to address/fix this in future.